rllib/examples/rl_modules/classes/vpg_rlm.py

```diff
@@ -1,9 +1,7 @@
+import torch
```

Comment:
No `try_import_torch` anymopre needed?

Reply:
Good point, but I think we can leave the bare-metal `import torch` everywhere that's not in the "common" import path (and thus doesn't break anything for non-torch users).